src/components/DogBattle.js

```diff
@@ -3,13 +3,42 @@ import Dog from "./Dog.js";
 import "./DogBattle.css";
 
 class DogBattle extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      DogBattle: [],
```

Comment:
I think you can use DogBattle as String rather than Array as you are getting one random dog at a time. Am I right ? Could you please give us any suggestion @NateWr ?  The code is working properly..

Reply:
This looks great, @swazz! Good work! I'd recommend you merge this and move forward.

If you want to improve the code, here's a suggestion. You can use just one state key, `DogBattle`, instead of two, `DogBattle` and `DogBattle1`. It would be an array with two items, one for each battle.

Consider how you might change things if the state was initialized like this:

```
this.state = {
  DogBattle: [
    'http://via.placeholder.com/300x300',
    'http://via.placeholder.com/300x300',
  ],
};
```